### ISSUE TITLE
ods-charts truncated in mobile devices

### ISSUE THREAD

**Issue by scMarth** [opendatasoft/ods-widgets#113]
When I create ods-charts and view them in mobile devices, some may look fine in landscape mode but in portrait mode, they get truncated.

Is this a limitation of ods-chart or is there a way around this?

**Comment by fpassaniti** [comment]
May you provide some screenshots please ? and in which display mode you encounter the issue ?

thanks

**Comment by scMarth** [comment]
Here is an example. These are two screenshots taken from a physical Android device. The first picture is in landscape mode. In this picture, it looks fine:

![screenshot_20180723-082534](https://user-images.githubusercontent.com/22603557/43086787-0f7d51c4-8e53-11e8-8188-6f76b971116e.png)

However, when the same chart is viewed in portrait mode, the chart looks truncated:

![screenshot_20180723-082517](https://user-images.githubusercontent.com/22603557/43086853-3aee304e-8e53-11e8-8368-0b26a9fd8436.png)

**Comment by fpassaniti** [comment]
Indeed, and as it's a technical limitation of the widget library we are using, we don't have any other advice than forcing a specific minimum width, to enable an horizontal scrollbar. I think it's acceptable on mobile display with the touch navigation.
